src/applications/simple-forms/26-4555/config/form.js

```diff
@@ -48,20 +48,24 @@ const formConfig = {
     // },
   },
   version: 0,
-  prefillEnabled: true,
+  migrations: [],
+  prefillEnabled: false,
+  // disableSave: false,
   savedFormMessages: {
     notFound: 'Please start over to apply for adapted housing.',
     noAuth:
       'Please sign in again to continue your application for adapted housing.',
   },
-  title:
-    'Apply for a Specially Adapted Housing Grant or Special Home Adaptation Grant',
+  title: 'Apply for a Specially Adapted Housing Grant Grant',
+  subTitle: 'VA Form 26-4555',
   defaultDefinitions: {
```

Comment:
You should be able to make this line simply: `defaultDefinitions: definitions,` 

and rewrite the `commonDefinitions` import line to be: `import { definitions } from 'vets-json-schema/dist/definitions.json';`

I think that should work. If there's any issues with that import, you could also do it how we did it in the 1010d branch: https://github.com/department-of-veterans-affairs/vets-website/blob/form/1010d/src/applications/1010d/config/form.js#L45

There we just import the whole `fullSchema` and then make the `defaultDefinitions` property: `defaultDefinitions: fullSchema.definitions,`

Reply:
Refactored as directed.